lib/degreed/content/courses.rb

```diff
@@ -28,12 +28,17 @@ def create(title:, external_id:, url:, duration:, duration_type:, **kwargs)
         request.post(
           content_courses_url,
           body: {
-            title: title,
-            summary: kwargs.fetch(:summary, nil),
-            "external-id": String(external_id),
-            url: url,
-            duration: Integer(duration),
-            "duration-type": duration_type
+            data: {
```

Comment:
I really botched this 😆 the docs clearly have the attributes nested like this: https://api.degreed.com/docs/#create-a-new-course

Reply:
Idk, the indentation in the docs is weird, I'm happy to make that the scapegoat.